ui/src/common/components/CitationSummaryGraph/CitationSummaryGraph.jsx

```diff
@@ -37,12 +45,102 @@ const xValueToLabel = {
   '250.0-500.0': '250-499',
   '500.0-*': '500+',
 };
+const rangeToQuery = new BiMap({
+  '0.0-1.0': '0--0',
+  '1.0-10.0': '1--9',
+  '10.0-50.0': '10--49',
+  '50.0-100.0': '50--99',
+  '100.0-250.0': '100--249',
+  '250.0-500.0': '250--499',
+  '500.0-*': '500--',
+});
+
+const typeToColors = {
+  Citeable: { color: BLUE, hoveredColor: HOVERED_BLUE },
+  Published: { color: ORANGE, hoveredColor: HOVERED_ORANGE },
+};
 
 class CitationSummaryGraph extends Component {
-  static toSeriesData(bucket) {
+  constructor() {
+    super();
+    this.state = {
+      hoveredBar: null,
+    };
+  }
+
+  getSelectedBar() {
+    const { query } = this.props;
+    if (query.citeable && query.citation_count) {
+      if (query.refereed) {
+        return {
+          selectedType: PUBLISHED,
```

Comment:
doesn't need to have `selected` prefix since the function name already indicates this is for `selectedBar`